myhdl/_traceSignals.py

```diff
@@ -65,6 +66,7 @@ def __init__(self):
         self.filename = None
         self.timescale = "1ns"
         self.tracelists = True
+        self.tracebackup = True
```

Comment:
I think this should be False by default to avoid breaking people's workflows.

Reply:
Which parameter are you talking about ?

Reply:
Just realized you're talking about self.tracebackup
I added this parameter because traces are always backed-up. As I don't want/need this behavior, I added the possibility to not backup the traces and I set this parameter to True by default to mimic current behavior.